docs/website/docs/plus/getting-started/tutorial.md

```diff
@@ -0,0 +1,395 @@
+* The `runtime` section is analogous to the config.toml [runtime] section and could also be omitted in this case.
+* The `profiles` section is not doing much in this case. There are two implicit profiles: `dev` and `tests` that are present in any project; we will learn about profiles in more detail later.
+
+You can reference environment variables in the `dlt.yml` file using the `${ENV_VARIABLE_NAME}` syntax. Additionally, dlt+ provides several [predefined project variables](../features/projects.md#substitution) that are automatically substituted during loading.
+
+:::tip
+You can find more information about the `dlt.yml` structure in the [dlt+ Project section](../core-concepts/project.md).
+:::
+
+## Running the pipeline
+
+Once the project is initialized, you can run the pipeline using:
+
+```sh
+dlt pipeline my_pipeline run
+```
+
+This command:
+- Locates the pipeline named `my_pipeline` in `dlt.yml`.
+- Executes it, populating the duckdb destination that is defined to be stored in `${tmp_dir}my_data.duckdb`.
+
+:::tip
+Take a look at the [Projects context](../features/projects.md#project-context) to learn more about how to work with nested projects and how dlt searches for the pipelines based on its name.
+:::
+
+### Inspecting the results
+
+Use the `dlt dataset` command to interact with the dataset stored in the DuckDB destination. For example:
+
+### Counting the loaded rows
+To count rows in the dataset, run:
+
+```sh
+dlt dataset my_pipeline_dataset row-counts
+```
+
+This will show the number of rows in the items table as specified by the arrow source. Additionally, the internal dlt tables are shown.
+
+```sh
+            table_name  row_count
+0                items        100
+1         _dlt_version          1
+2           _dlt_loads          1
+3  _dlt_pipeline_state          1
+```
+
+### View data
+To view the first five rows of the `items` table:
+
+```sh
+dlt dataset my_pipeline_dataset head items
+```
+
+This displays the top entries in the `items` table, enabling quick validation of the pipeline's output. The output will be something like this:
+
+```sh
+Loading first 5 rows of table items.
+
+   id   name  age
+0   0  jerry   49
+1   1    jim   25
+2   2   jane   46
+3   3   john   48
+4   4  jenny   49
+```
+
+To show more rows, use the `--limit` flag.
+
+```sh
+dlt dataset duckdb_dataset head items --limit 50
+```
+
+## Adding sources, destinations, and pipelines to your project
+
+Adding a new entity to an existing dlt+ Project is easy. You can add a new entity to your project by running the `dlt <entity_type> <entity_name> add` command. Depending on the entity you are adding, different options are available. To see all options for adding a destination, for example, you can run `dlt destination add --help`. Let's individually add a source, destination, and pipeline to a new project, replicating the default project we created in the previous chapter.
```

Comment:
link to cli ref?

Reply:
added in https://github.com/dlt-hub/dlt/pull/2325